content/en/docs/tasks/administer-cluster/migrating-from-dockershim/troubleshooting-cni-plugin-related-errors.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Troubleshooting CNI plugin-related errors
+content_type: task
+reviewers:
+- mikebrow
+weight: 10
+---
+
+<!-- overview -->
+
+To avoid CNI plugin-related errors, verify that you are using or upgrading to a
+container runtime that has been tested to work correctly with your version of
+Kubernetes.
+
+For example, the following container runtimes have been tested with Kubernetes
+1.24:
+
+* containerd v1.6.4 and later, v1.5.11 and later
+* CRI-O 1.24 and later
```

Comment:
Not sure this z-stream distinction is important in this context but it was called out by @haircommander:
```suggestion
* CRI-O 1.24.0 and later
```

Reply:
Updated. Thanks.